packages/iTwinUI-react/src/core/Tabs/Tabs.tsx

```diff
@@ -94,6 +111,10 @@ export type VerticalTabsProps = Omit<TabsProps, 'orientation' | 'type'> & {
  *   <Tab label='Label 2' icon={<SvgPlaceholder />} />,
  * ];
  * <Tabs labels={tabsWithIcons} type='pill' />
+ *
+ * @example
+ * // Vertical Tabs
+ * <Tabs orientation='vertical' labels={tabs} />
```

Comment:
Since Tabs was missing `@example` with "vertical" Tabs

Reply:
Thanks, I removed the `// comment`. I thought of replacing it with a `<caption>`, but it looked out of place since other examples didn't have captions. So didn't add a caption.

I also moved the example to the top on [L101-102](https://github.com/iTwin/iTwinUI-react/blob/3ed1ceafda3e293a226d2363a251c4ff8ca44c74/packages/iTwinUI-react/src/core/Tabs/Tabs.tsx#L101-L102) so that it is near the horizontal tabs example.